src/resources/billing.js

```diff
@@ -0,0 +1,100 @@
+import T from 'proptypes';
+
+export const propTypes = {
+  id: T.string,
+  disabled_at: T.string,
+  object: T.string,
+  name: T.string,
+  last4: T.string,
+  expiration_month: T.string,
+  expiration_year: T.string,
+  brand: T.string,
+  country: T.string,
+  verified: T.bool,
+  bank_account: T.string,
+};
+
+export default (api) =>
+  class Billing {
+    static _name = 'Billing';
+
+    static _url = 'billing';
+
+    static key = 'billing';
+
+    static propTypes = propTypes;
+
+    /**
+     * fund your EasyPost wallet by charging your primary or secondary payment method on file.
+     * @param {String} amount
+     * @param {String} primaryOrSecondary
+     * @returns {Promise<never>}
+     */
+    static async fundWallet(amount, primaryOrSecondary = 'primary') {
+      const paymentInfo = await this.getPaymentInfo(primaryOrSecondary.toLowerCase());
+      const endpoint = paymentInfo[0];
+      const paymentMethodID = paymentInfo[1];
+      const wrappedParams = { amount };
+
+      const res = await api.post(`${endpoint}/${paymentMethodID}/charges`, wrappedParams);
+      return res.body;
+    }
+
+    /**
+     * delete a payment method
+     * @returns {Promise|Promise<never>}
+     */
+    static async deletePaymentMethod(primaryOrSecondary) {
+      const paymentInfo = await this.getPaymentInfo(primaryOrSecondary.toLowerCase());
+      const endpoint = paymentInfo[0];
+      const paymentMethodID = paymentInfo[1];
+
+      const res = await api.del(`${endpoint}/${paymentMethodID}`);
+      return res.body;
```

Comment:
Issue: This returns nothing, I believe we determined we were going to return `true` if successful?